Add activestate OIDC docs (#15548)
pypi/warehouse: docs/user/trusted-publishers/creating-a-project-through-oidc.md

=== "ActiveState"

    Setting up ActiveState to create a PyPI project is the same as

>     updating a project. You need to provide the project name on ActiveState's Platform,

Reviewer: ```suggestion
    updating a project. You need to provide the name of the ActiveState project,
```

>     the ActiveState organization that project belongs to, and the username who will be performing

Reviewer: ```suggestion
    the ActiveState organization that project belongs to, and the ActiveState user who will be performing
```
